client/lib/translator-jumpstart/index.js

```diff
@@ -84,7 +84,15 @@ communityTranslatorJumpstart = {
 			optionsFromPage = {};
 		}
 
-		props = { className: 'translatable' };
+		if ( 'string' !== typeof originalFromPage ) {
+			debug( 'unknown original format' );
+			return displayedTranslationFromPage;
+		}
+
+		const props = {
+			className: 'translatable',
+			'data-singular': originalFromPage
```

Comment:
I am not sure we can do this. This is only when the page provides the original and we don't query the translate.wordpress.com API for the string, right?